### PR TITLE
NORA-B1 updates for NCS v2.5.0

### PR DESCRIPTION
**Pull request by ghost**
Moved old EVK-NORA-B1 and MINI-NORA-B1 folders to an archive folder
Added new EVK-NORA-B10, EVK-NORA-B12, MINI-NORA-B10, and MINI-NORA-B12 folders, tested with NCS v2.5.0. Files follow nrf5340DK_nrf5340 file naming conventions.

### CODE REVIEW

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_cpuapp_common.dtsi [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+			   <3 0 &gpio0 7 0>,	/* A3 */
+			   <4 0 &gpio0 25 0>,	/* A4 */
+			   <5 0 &gpio0 26 0>,	/* A5 */
+			   <6 0 &gpio1 0 0>,	/* D0 */
+			   <7 0 &gpio1 1 0>,	/* D1 */
+			   <8 0 &gpio1 4 0>,	/* D2 */
+			   <9 0 &gpio1 5 0>,	/* D3 */
+			   <10 0 &gpio1 6 0>,	/* D4 */
+			   <11 0 &gpio1 7 0>,	/* D5 */
+/*			   <12 0 &gpio1 8 0>,	   D6 FEM CTX */
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_cpuapp_common.dtsi:92]: Do we have FEM pins also on NORA-B10?

Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_cpuapp_common.dtsi]: Do we define FEM pins also on NORA-B10?

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_cpuapp_common.dtsi [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+				 <5 &adc 5>;	/* A5 = P0.26 = AIN5 */
+	};
+
+	gpio_fwd: nrf-gpio-forwarder {
+		compatible = "nordic,nrf-gpio-forwarder";
+		status = "okay";
+		uart {
+			gpios = <&gpio1 1 0>, <&gpio1 0 0>, <&gpio0 11 0>, <&gpio0 10 0>;
+		};
+		/* Forward pins for NORA-B10 FEM */
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_cpuapp_common.dtsi]: Do we define FEM pins also on NORA-B10?

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts [hunk 3]
```diff
@@ -82,8 +89,8 @@
 			   <9 0 &gpio1 5 0>,	/* D3 */
 			   <10 0 &gpio1 6 0>,	/* D4 */
 			   <11 0 &gpio1 7 0>,	/* D5 */
-			   <12 0 &gpio1 8 0>,	/* D6 */
-			   <13 0 &gpio1 9 0>,	/* D7 */
+/*			   <12 0 &gpio1 8 0>,	   D6 FEM CTX */
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts]: FEM on NORA-B10?

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts [hunk 4]
```diff
@@ -109,6 +116,14 @@
 		mcuboot-led0 = &led0;
 		watchdog0 = &wdt0;
 	};
+	/* Enable the NORA-B10 FEM */
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts]: FEM on NORA-B10?

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet_defconfig [hunk 5]
```diff
@@ -24,3 +24,9 @@ CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
 CONFIG_PINCTRL=y
+
+# enable NORA-B10 fem
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet_defconfig]: FEM?

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_cpuapp_common.dtsi [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+		};
+	};
+
+	gpio_fwd: nrf-gpio-forwarder {
+		compatible = "nordic,nrf-gpio-forwarder";
+		status = "okay";
+		uart {
+			gpios = <&gpio1 1 0>, <&gpio1 0 0>, <&gpio0 11 0>, <&gpio0 10 0>;
+		};
+		/* Forward pins for NORA-B10 FEM */
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_cpuapp_common.dtsi]: FEM on B10?

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts [hunk 7]
```diff
@@ -1,55 +1,61 @@
...
+			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
 			label = "Blue LED 2";
 		};
 	};
 
 	buttons {
 		compatible = "gpio-keys";
 		button0: button_0 {
-			gpios = <&gpio1 7 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			gpios = <&gpio0 22 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts]: Should GPIO 22 be button 1 and GPIO 4 button 0?

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts [hunk 8]
```diff
@@ -60,7 +66,18 @@
 		sw0 = &button0;
 		sw1 = &button1;
 		bootloader-led0 = &led0;
+		mcuboot-button0 = &button0;
+		mcuboot-led0 = &led0;
+		watchdog0 = &wdt0;
 	};
+	/* Enable the NORA-B10 FEM */
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts]: FEM?

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet_defconfig [hunk 9]
```diff
@@ -22,3 +22,11 @@ CONFIG_SERIAL=y
 # enable console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+CONFIG_PINCTRL=y
+
+# enable NORA-B10 fem
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet_defconfig]: FEM?

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet-pinctrl.dtsi [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+		group2 {
+			psels = <NRF_PSEL(UART_RX, 0, 20)>,
+				<NRF_PSEL(UART_CTS, 0, 21)>;
+			bias-pull-up;
+		};
+	};
+
+	uart0_sleep: uart0_sleep {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 23)>,
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet-pinctrl.dtsi]: Pls double check these pins, I am a bit uncertain on the definition.

File: zephyr/boards/arm/ubx_mininorab12_nrf5340/ubx_mininorab12_nrf5340_cpunet-pinctrl.dtsi [hunk 11]
```diff
@@ -0,0 +1,63 @@
+/*
+ * u-blox MINI-NORA-B12 board configuration
+ *
+ * Copyright (c) 2023 u-blox AG
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
```
Comment (perssonmagnus) [inline zephyr/boards/arm/ubx_mininorab12_nrf5340/ubx_mininorab12_nrf5340_cpunet-pinctrl.dtsi:10]: Pls double check pin numbers, not sure if I understand how to define this.

**Review by perssonmagnus — COMMENTED** [review]
No doc folder for mini-NORA-B1 boards?

Not new, but I think the 32 MHz is not external to the module, right? https://github.com/u-blox/u-blox-sho-OpenCPU/blob/fe62a574ff37140ea96fb09ebf4de8b2176a7251/zephyr/boards/arm/ubx_evknorab10_nrf5340/doc/index.rst?plain=1#L60

**Comment by ghost** [comment]
> No doc folder for mini-NORA-B1 boards?
> 
> Not new, but I think the 32 MHz is not external to the module, right?

I reworded the clock section, and added doc folders for MINI-NORA-B1 boards.

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_cpuapp_common.dtsi (hunk 2 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_cpuapp_common.dtsi]: corrected

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts (hunk 3 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts]: corrected

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts (hunk 4 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet.dts]: corrected

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_cpuapp_common.dtsi (hunk 6 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_cpuapp_common.dtsi]: corrected

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts (hunk 7 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts]: corrected

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet_defconfig (hunk 9 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet_defconfig]: corrected

File: zephyr/boards/arm/ubx_mininorab12_nrf5340/ubx_mininorab12_nrf5340_cpunet-pinctrl.dtsi (hunk 11 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_mininorab12_nrf5340/ubx_mininorab12_nrf5340_cpunet-pinctrl.dtsi:10]: Followed nrf5340dk_nrf5340 style. Pins were swapped, corrected now.

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet-pinctrl.dtsi (hunk 10 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet-pinctrl.dtsi]: Followed nrf5340dk_nrf5340 style.

File: zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts (hunk 8 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_mininorab10_nrf5340/ubx_mininorab10_nrf5340_cpunet.dts]: corrected

File: zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet_defconfig (hunk 5 above)
Comment (ghost) [reply zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpunet_defconfig]: corrected

**Review by perssonmagnus — APPROVED** [review]
Looks good to me!

**Pull request merged**